modules/repository/repo.go

```diff
@@ -240,16 +237,11 @@ func MigrateRepositoryGitData(ctx context.Context, u *user_model.User,
 
 // cleanUpMigrateGitConfig removes mirror info which prevents "push --all".
 // This also removes possible user credentials.
-func cleanUpMigrateGitConfig(configPath string) error {
-	cfg, err := ini.Load(configPath) // FIXME: the ini package doesn't really work with git config files
-	if err != nil {
-		return fmt.Errorf("open config file: %w", err)
-	}
-	cfg.DeleteSection("remote \"origin\"")
-	if err = cfg.SaveToIndent(configPath, "\t"); err != nil {
-		return fmt.Errorf("save config file: %w", err)
-	}
-	return nil
+func cleanUpMigrateGitConfig(ctx context.Context, repoPath string) error {
+	cmd := git.NewCommand(ctx, "remote", "rm", "origin")
+	return cmd.Run(&git.RunOpts{
```

Comment:
I think it should tolerate if the "origin" doesn't exist?

eg:

```
~/work/gitea$ git remote rm no-such                                                                                                                                                                                                  
error: No such remote: 'no-such'
~/work/gitea$ echo $?
2
~/work/gitea$
```

Reply:
Done.